src/components/layout/layout.js

```diff
@@ -25,18 +25,15 @@ const has_dataLayer = isBrowser() && window.dataLayer
 const TRACKING_STATUS_KEY = 'tracking_status'
 const tracking_status_cookie = new CookieStorage(TRACKING_STATUS_KEY)
 
-const cfd_warning_height_desktop = 8
-const cfd_warning_height_tablet = 12
-
 const CFDWrapper = styled.section`
     background-color: var(--color-grey-25);
     background-size: cover;
-    height: ${cfd_warning_height_desktop}rem;
+    height: ${cfd_warning_height.desktop}rem;
     display: flex;
     align-items: center;
 
     @media ${device.tabletS} {
-        height: ${cfd_warning_height_tablet}rem;
+        height: ${cfd_warning_height.tablet}rem;
```

Comment:
<img width="452" alt="Screenshot 2021-01-26 at 10 37 07 AM" src="https://user-images.githubusercontent.com/75345074/105792537-98fddc00-5fc2-11eb-948d-21e3bce7d342.png">

rem units gives diff px values in mobile. if this value px value is the desired one then it should be ok. if its not, my suggestion is using px instead of rem

`${cfd_warning_height.tablet * 10}px;`

Reply:
layout offset was calculated in rem, that's why need to use rem.